base/src/main/java/com/smartdevicelink/proxy/rpc/SystemCapability.java

```diff
@@ -49,6 +50,7 @@
 	public static final String KEY_VIDEO_STREAMING_CAPABILITY = "videoStreamingCapability";
```

Comment:
We can change the type in the `getCapabilityForType()`/`setCapabilityForType()` to return/accept java `Object` instead of `RPCStruct` so they can use `List<DisplayCapability>`. That shouldn't be a breaking change because `Object` is a more generic type than `RPCStruct` so existing code that uses these APIs should still work in most cases.

Reply:
Also, could you please add Javadoc for `getCapabilityForType(), setCapabilityForType()`?